src/Tools/SignDynamo/SignDynamo.csproj

```diff
@@ -9,8 +9,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>SignDynamo</RootNamespace>
     <AssemblyName>SignDynamo</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <TargetFrameworkProfile />
```

Comment:
Same question as before, is this required? I see no harm in it being here, more of a question as to why.